### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by Xnyle** [fennifith/Alarmio#126]
Would be nice to have the option to auto dismiss an alarm after given time.

**Comment by kentone** [comment]
I was about to add this one :open_mouth: :+1:

**Comment by test2a** [comment]
yes. i had a morning alarm with an internet stream going on. i didn't wake up to close it and it just ran and ran until the phone got hot and the battery died.
